governance.md

```diff
@@ -0,0 +1,147 @@
+
+The Core Team will conduct a regular review of each software project, annually for core projects and every two years for others. Additional review may be called for if a project is having problems. The review should look at releases, bug fixing and other indications that the project is performing as desired.
+
+The Core Team is called upon to resolve conflicts between software projects or if a consensus cannot be reached on an important decision within a single project. Individual projects may also refer important decisions to the Core Team, particularly those that reflect our community values. Issues of copyright and licensing are always referred to the core team.
+
+The Core Team is responsible for approving, maintaining and amending these governance policies as needed.
+
+Membership in the Core Team is by invitation of the existing members. A nomination will result in discussion and possibly a vote by the existing members. As with all its decisions, the Core Team will attempt to reach a consensus on new members. If a vote is required, the new member must be approved by 2/3 of the membership.
+
+Core Team members remain on the team until they choose to retire or until 2/3 of the members vote to remove them.
+
+The initial composition of the Core Team is
+* Charlie Poole
+* Rob Prouse
+* Chris Maddock
+* Terje Sandstrom
+* Jamie Cansdale
+
+### Core Team Lead
+
+The Core Team Lead is a single individual, voted for by the Core Team members. Once someone has been appointed Lead, they remain in that role until they choose to retire, or the Core Team casts a two-thirds majority vote to remove them.
+
+The Lead has no additional authority over other members of the Core Team: the role is that of a coordinator and facilitator. The Chair is also expected to ensure that all governance processes are adhered to, and has the casting vote when the project fails to reach consensus.
+
+>**Reviewers:** If you like Organization Lead better, that's OK with me too. I considered Community Leader as well, but it seemed to overstate the case. The role is more like the go-between from the Core Team to the Community.
+
+### Original Contributor
+
+The .NET Foundation, which we are considering joining, defines the role of Original Contributor as the person who seeds the project with the original software base. For the purposes of NUnit - at least NUnit 3 - that individual is identified as Charlie Poole who is the copyright holder for the core NUnit software components until they are assigned or licensed elsewhere.
+
+>**Reviewers:** It's interesting that nobody commented on this in the early reviews. This is me wanting to be sure of the direction (initial at least) of things before letting go entirely. I have a lot of confidence in all of you, and I don't __really__ expect to exercise any authority in this role.
```

Comment:
I think that we (or maybe more you and I) have had enough conversations about this that it isn't an issue. The .NET Foundation also has a similar term to the Original Contributor, so if we are moving there, then it is a good thing to specify. I would have preferred Benevolent Dictator, but up to you 😉